client/sand/http.go

```diff
@@ -27,12 +29,15 @@ func (c *client) rawDialer(ctx context.Context, sandNetworkID, network, address
 }
 
 type HTTPRoundTripperOpts struct {
-	TLSConfig *tls.Config
+	TLSConfig         *tls.Config
+	DisableKeepAlives bool
```

Comment:
Why adding `DisableKeepAlives` this as opts? We want to disable it completely and not make this optional

Reply:
Keepalived is not useful in our case because we don't keep a pool of connection, so disable it by default is the right way